src/config/graphical-session/kde.md

```diff
@@ -17,3 +17,23 @@ instead.
 If you wish to start an X-based session from the console, use
 [startx](./xorg.md#startx) to run `startplasma-x11`. For a Wayland-based
 session, run `startplasma-wayland` directly.
+
+## Dolphin
+
+Dolphin is the default file manager of the KDE desktop environment. It can be
+installed on its own by installing the `dolphin` package, or it can be installed
+as part of the `kde5` or `kde5-baseapps` packages.
+
+### Thumbnail Previews
+
+To have properly working thumbnail file previews, the packages
+`ffmpegthumbnailer` and `kdegraphics-thumbnailers` have to be installed. Enable
+previews in "Control" -> "Configure Dolphin" -> "General" -> "Previews" by
+checking the corresponding boxes. File previews will be shown for the selected
+file types after clicking "Preview" in dolphin's toolbar.
+
+### Integrated Terminal
+
+The integrated terminal function (key F4) opens a terminal emulator inside
+dolphin's window, on the same directory being currently browsed. It is
+configured to require Konsole (KDE's default terminal emulator) by default.
```

Comment:
I don't think this part is very Void specific, what do you think? Not sure we should add it.

Reply:
You're right, I just moved it over because it was in the wiki. It's been removed.